hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleSchemaRegistry.java

```diff
@@ -277,12 +286,13 @@ && alreadyIncludesStateDefs(previousVersion, s.getVersion()))
     private RedefinedWritableStates applyStateDefinitions(
             @NonNull final Schema schema,
             @NonNull final List<Schema> schemasAlreadyInState,
-            @NonNull final Configuration configuration,
+            @NonNull final Configuration nodeConfiguration,
+            @NonNull final Configuration platformConfiguration,
```

Comment:
What's the difference between node and platform configs? Shouldn't they be the same? My understanding was `Configuration` is just a holder that can be used to get various configs: MerkleDbConfig, ConsensusConfig, and so on.

Reply:
- `nodeConfiguration` - you can see it's initialization [here](https://github.com/hashgraph/hedera-services/blob/a7797da24d01b49b29c8509b8f519690a23d5918/hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java#L482-L482). I took this variable name because of [this field's docs](https://github.com/hashgraph/hedera-services/blob/a7797da24d01b49b29c8509b8f519690a23d5918/hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java#L248-L248). It does not contain `MerkleDbConfig`, etc.
- `platformConfiguration` - its the configuration which contains `MerkleDbConfig`, etc. It actually supersedes configuration in the `ConfigurationHolder` and is passed down to all classes that need the relevant configuration. You can see its initialization [here](https://github.com/hashgraph/hedera-services/blob/a7797da24d01b49b29c8509b8f519690a23d5918/hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java#L202-L202). Its called this way, because this configuration is used in the platform context initialization [here](https://github.com/hashgraph/hedera-services/blob/a7797da24d01b49b29c8509b8f519690a23d5918/hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java#L234-L234).